### ISSUE TITLE
How to manage fields and alternatives ?

### ISSUE THREAD

**Issue by ikus060** [lektor/lektor#391]
Hello,

We are using Lektor to generate a web site with two alternatives: French is the default, English is the alternative. While exploring the feature, I realize it's possible to declare common fields value into `content.lr` and declare language specific value into content+ALT.lr.
This is a very nice feature, but I would like to "help" my user by restricting which fields is language specific and which fields are common. Is it something possible to do with the current version of Lektor ?

I'm looking from something like this to be available in the model `.ini` file:
```
[fields.title]
label = Title
type = string
alt_enabled=true

[fields.width]
label = Picture Width
type = integer
alt_enabled=false
```

Basically, `title` need an alternative for French and English, while `width` cleary doesn't need to translated and should be common to all alternative.

**Comment by nixjdm** [comment]
I haven't used alternatives much, but I'm pretty sure this feature doesn't exist right now. It sounds useful.

**Comment by nixjdm** [comment]
I think I like something a bit more general like `alts_enabled` defaulting to true or all, and optionally specifying a list of acceptable alts. E.g.
```
alts_enabled = en, de, fr
```
which may omit `es`.

Edit: Actually, though this is more general I'm struggling to think of a use case for it being this general. Perhaps the binary toggle is really all we need.

**Comment by nixjdm** [comment]
Note: The docs for alts in https://github.com/lektor/lektor-website definitely need updated when this is done.

**Comment by ikus060** [comment]
Here is a first implementation. I'm willing to change the code following your recommendation.

**Comment by ikus060** [comment]
Hello, is it possible to get feedback for my changes ? 

Thanks

**Comment by nixjdm** [comment]
Hey @ikus060, sorry for taking a while. I just put a minor edit in your pull request because the linter was complaining and pulled it after it passed the tests. It looks really good to me. It was very straight forward.

I also made https://github.com/lektor/lektor-website/issues/156 for updating the docs to describe this new feature. Please take a look at that. :)

**Comment by dairiki** [comment]
Note that this was implemented in PR #408
